content/theme/characters/archetypes/jedi/jedi-consular.md

```diff
@@ -4,39 +4,41 @@ date: 2020-11-09T21:18:25-08:00
 draft: false
 layout: archetype
 attributes:
-  awareness: 9
+  awareness: 7
   coordination: 7
-  knowledge: 10
+  knowledge: 9
   physique: 7
   presence: 11
-  reflex: 11
+  reflex: 8
+  force: 8
```

Comment:
This is fine for now, but for the Jedi archetypes we probably want them to be at Knight level (3D+0 Force), because I don't think a Padawan can really be that focused until then? Open to discussion though.

Reply:
Oh, yeah I agree with that. I wasn't sure what kind of number should be there. So that's at least 9 right?